src/repart/repart.c

```diff
@@ -10370,7 +10370,6 @@ static int vl_method_run(
                                 need_free > context->total ? "io.systemd.Repart.DiskTooSmall" :
                                                              "io.systemd.Repart.InsufficientFreeSpace",
                                 JSON_BUILD_PAIR_UNSIGNED_NOT_EQUAL("minimalSizeBytes", size, UINT64_MAX),
-                                JSON_BUILD_PAIR_UNSIGNED_NOT_EQUAL("needFreeBytes", need_free, UINT64_MAX),
```

Comment:
I don't think we can drop this, since -rc1 was already made and this would be a compat break. @poettering?